src/contract-details/SellAtMarketButton.js

```diff
@@ -20,7 +20,7 @@ export default class SellAtMarketButton extends Component {
 				className="sell-at-market-btn"
 				onClick={onClick}
 			>
-				<M m="Sell at Market" />
+				<M m="Sell at Market " />
```

Comment:
Because there will be price next to this span, and it does not looks nice without spacing